content/docs/command-reference/remote/modify.md

```diff
@@ -616,8 +621,8 @@ more information.
   [delegated] to the service account if needed.
 
   ```dvc
-  $ dvc remote modify myremote \
-                 gdrive_service_account_user_email 'myemail-addr'
+  $ dvc remote modify --local myremote \
```

Comment:
this is a very good example when actual changes and refactoring (changing `--local` location) could go in two different PRs. 

This is PR is supposed to be quick and easy, but requires time to review properly.